tools/pylint-extensions/azure-pylint-guidelines-checker/README.md

```diff
@@ -87,3 +87,4 @@ docstring-missing-type | pylint:disable=docstring-missing-type    |       Docstr
 docstring-missing-return | pylint:disable=docstring-missing-return    |       Docstring missing return.                        | [link](https://azure.github.io/azure-sdk/python_documentation.html#docstrings)           |
 docstring-missing-rtype | pylint:disable=docstring-missing-rtype    |       Docstring missing return type.                        | [link](https://azure.github.io/azure-sdk/python_documentation.html#docstrings)           |
 docstring-should-be-keyword | pylint:disable=docstring-should-be-keyword    |       Docstring should use keywords.                        | [link](https://azure.github.io/azure-sdk/python_documentation.html#docstrings)           |
+no-azure-core-http-response-import | pylint:disable=no-azure-core-http-response-import    |      Do not import HttpResponse from Azure Core.                       | [link](https://github.com/Azure/azure-sdk-for-python/issues/30785)           |
```

Comment:
Maybe let's re-word to something like "no legacy import". Otherwise this makes me think I should never import HttpResponse from azure.core